packages/semver/src/executors/version/testing.ts

```diff
@@ -47,21 +47,24 @@ export function setupTestingWorkspace(
 }
 
 export function createFakeContext({
+  cwd = process.cwd(),
   project,
   projectRoot,
   workspaceRoot,
   additionalProjects = [],
 }: {
+  cwd?: string;
   project: string;
   projectRoot: string;
   workspaceRoot: string;
   additionalProjects?: {
     project: string;
     projectRoot: string;
-    targets?: Record<string, Target>;
+    targets?: Record<string, TargetConfiguration>;
```

Comment:
`Target` does not match the `ExecutionContext` interface
Changing this to `TargetConfiguration` does require a bit more configuration during the setup of additionalProjects but it keeps things type safe.

Reply:
Would you create an issue for this?